### ISSUE TITLE
[make:entity] Issue with nullable json field

### ISSUE THREAD

**Issue by apphancer** [symfony/maker-bundle#1298]
When creating a field using the json type, the command asks if this field can be nullable. When answering yes to this question, the following code is generated for the Entity:

```php
#[ORM\Column(nullable: true)]
private array $jsonFieldData = [];
```

However, when this entity is used (e.g. using a find method from its repository) this seems to be causing the following error:

Cannot assign null to property App\Entity\EntityName::$jsonFieldData of type array

I seem to be getting this to work if I manually change the default value for the property from an empty array to null and making the property type nullable.

```php
#[ORM\Column(nullable: true)]
private ?array $jsonFieldData = null;
```

symfony/maker-bundle v1.48.0
doctrine/doctrine-bundle v2.7.1
doctrine/orm v2.13.

**Comment by ClemRiviere** [comment]
Seems to be fixed with symfony/maker-bundle v1.57.0

**Comment by jrushlow** [comment]
fixed in #1293 - released in `v1.50.0`
